generators/server/templates/src/main/resources/_logback-spring.xml

```diff
@@ -61,7 +61,7 @@
         <resetJUL>true</resetJUL>
     </contextListener>
 
-    <root level="@logback.loglevel@">
+    <root level="#logback.loglevel#">
```

Comment:
The logback config only determines its initial state, the GUI changes this dynamically (but doesn't write back to the config file, obviously. I haven't made a point of testing it explicitly, though.